tests/testthat/test-derive_param_extreme_event.R

```diff
@@ -168,3 +169,43 @@ test_that("derive_param_extreme_event Test 3: latest evaluable tumor assessment
     keys = c("USUBJID", "PARAMCD", "ADT")
   )
 })
+
+## Test 4:
+## derive latest evaluable tumor assessment date parameter without overwriting existing result ----
+test_that("derive_param_extreme_event Test 4: latest evaluable tumor assessment date parameter", {
```

Comment:
Please format according to the unit test guidance (https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#set-up-the-test-script). You can achieve this easily by using the ["Format test_that test file" addin](https://pharmaverse.github.io/admiraldev/devel/articles/unit_test_guidance.html#addin-admiraldevformat_test_that_file).

Reply:
Thanks for the review @bundfussr!! I will update and re request for a review!